README.md

```diff
@@ -190,7 +190,7 @@ $ git checkout -b feature/foo develop
 
 You should use `feature/foo` to implement and commit all changed required for your new feature.
 
-* Make many small commits so that the history of development for you feature branch is clear and so that it is easy to pinpoint and edit or cherry-pick specific commits if necessary.
+* Make many small commits so that the history of development for you feature branch is clear and so that it is easy to pinpoint and edit or cherry-pick specific commits if neccessary.
```

Comment:
Thanks for your contribution! One thing though, I think 'necessary' is the correct form of spelling, as seen [here](https://dictionary.cambridge.org/dictionary/english/necessary). Could you update your PR? :)

Reply:
Done, thanks!